doc/installation_beginner.rst

```diff
@@ -139,7 +139,11 @@ You should see two compilers like ``x86_64-apple-darwin13.4.0-clang`` and ``x86_
 
 - type ``export CC=x86_64-apple-darwin13.4.0-clang`` and ``export CXX=x86_64-apple-darwin13.4.0-clang++``
 
+If not using Anaconda (in Linux OS)
+---------------------
 
+- You need to install gcc-7 using this `reference <https://forums.linuxmint.com/viewtopic.php?p=1465419&sid=72631feb0aa7bc551f199c606aecf256#p1465419>`_.
```

Comment:
I think we don't need to be specific about the `gcc-7`, but this example is good.

Maybe we rephrase it so users need gcc (4.9.x+? with C++14 support).

Reply:
I agree we don't need to be specific about version of gcc.